numpy/__init__.pyi

```diff
@@ -2908,9 +2908,9 @@ class floating(inexact[_NBit1]):
     ) -> float: ...
     def tolist(self) -> float: ...
     def is_integer(self) -> bool: ...
-    def hex(self: float64) -> str: ...
+    def hex(self: _FloatType) -> str: ...
```

Comment:
```suggestion
    def hex(self) -> str: ...
```
No need to use a typevar here, as you're dealing with <2 variables here.
`fromhex` looks fine though.

Reply:
Thanks, fixed in f48f0fc